docs/community-clients/index.asciidoc

```diff
@@ -43,6 +43,9 @@ a number of clients that have been contributed by the community for various lang
 [[clojure]]
 == Clojure
 
+* https://github.com/mpenet/spandex[Spandex]: 
+  Clojure client built on new/recent official Client
```

Comment:
Could you elaborate what `new/recent` means? Looking at the project page, it states that its a "low level rest-client wrapper". Maybe this should be mentioned here as well or instead.

Reply:
It means it (tries to) track the latest versions of the "new low level rest client". 
Sure I can update the wording.